src/Http/Http.Extensions/gen/StaticRouteHandlerModel/Endpoint.cs

```diff
@@ -108,8 +108,13 @@ public Endpoint(IInvocationOperation operation, WellKnownTypes wellKnownTypes, S
 
     public static bool SignatureEquals(Endpoint a, Endpoint b)
```

Comment:
Should the parameters be nullable?
```suggestion
    public static bool SignatureEquals(Endpoint? a, Endpoint? b)
```